SonarQube.TeamBuild.PostProcessor/Program.cs

```diff
@@ -16,12 +15,12 @@
 
 namespace SonarQube.TeamBuild.PostProcessor
 {
-    internal class Program
+    public class Program
```

Comment:
Reverted these (I was thinking of exposing them for testing but found a different way to test)